### ISSUE TITLE
[BUG] MFA without email server config

### ISSUE THREAD

**Issue by osmontero** [utmstack/UTMStack#117]
**Describe the bug**
Users should not be able to enable MFA without configuring an email server and verifying their email address.

**Expected behavior**
Users must verify their email address by clicking a verification link or entering a verification code sent by email. After this, they can enable multi-factor authentication (MFA) without risk of being locked out because it is not properly configured or their email address is incorrect.

**Environment**
 - OS: Ubuntu 22.04
 - Version v10.1.n

**Comment by jdieguez89** [comment]
@osmontero Good catch, @c3s4rfred @mjabascal10; let's implement a configuration check; when a configuration is saved, we need to check if all the prerequisites are met; for example, to save the TFA, the email configuration must be set. This will allow us to add more configurations that depend on each other. Let's discuss this.
